muswarmlogger/loggers/sparql/stats.py

```diff
@@ -37,8 +37,6 @@
     """
     if not event.status == "start":
         return
-    if not event.attributes.get('STATS'):
-        return
```

Comment:
It's there because not all containers are logged, only the ones that have a label STATS. Maybe you can improve the documentation for that because it's not clear.